pkg/scheduler/scheduler.go

```diff
@@ -109,6 +112,13 @@ func WithProfiles(p ...schedulerapi.KubeSchedulerProfile) Option {
 	}
 }
 
+// WithParallelism sets CPU Threads for Scheduler parallelize package. By default it is 16
```

Comment:
`WithParallelism sets the parallelism for all scheduler algorithms. Default is 16.`